src/components/offers/applyForm/ApplyForm.js

```diff
@@ -115,6 +127,7 @@ export const ApplyForm = () => {
                     }}
                     error={!validateSurname(surname) && shouldValidateField.surname}
                     helperText={!validateSurname(surname) && shouldValidateField.surname ? "Surname cannot be empty" : " "}
+                    disabled={loggedIn}
```

Comment:
Wydaje mi się, że powinniśmy udostępnić nawet zalogowanemu użytkownikowi zmianę danych przy aplikacji. Jest to już jego widzimi się, czy do nas zaloguje się z mailem 'imie.nazwisko@gmail.com', a chce zaaplikować na 'kociaczek.buziaczek@o2.pl' - powinien mieć taką możliwość, jeśli chce

Reply:
Takie mamy założenie, ale nie ma w tym momencie na to kompletnie miejsca w naszym modelu